website/project/views/node.py

```diff
@@ -727,6 +727,8 @@ def _view_project(node, auth, primary=False,
             'registered_meta': node.registered_meta,
             'registered_schemas': serialize_meta_schemas(list(node.registered_schema.all())) if is_registration else False,
             'is_fork': node.is_fork,
+            'is_collected': node.is_collected,
+            'collections': serialize_collections(node.linked_collections) if node.is_collected else [],
```

Comment:
>No point in doing this query twice here

Make that thrice. If you default to just returning an empty list from `linked_collections`, feeding an empty list into `serialize_collections` should return an empty list as well, obviating this ternary.